json/src/scheme/params.rs

```diff
@@ -93,15 +91,13 @@ mod tests {
             "minSetShardUsersCost" : 14,
             "minWrapCccCost" : 15,
             "minCustomCost" : 16,
-            "minStoreCost" : 17,
-            "minRemoveCost" : 18,
-            "minMintAssetCost" : 19,
-            "minTransferAssetCost" : 20,
-            "minChangeAssetSchemeCost" : 21,
-            "minComposeAssetCost" : 22,
-            "minDecomposeAssetCost" : 23,
-            "minUnwrapCccCost" : 24,
-            "minIncreaseAssetSupplyCost": 25,
+            "minMintAssetCost" : 17,
```

Comment:
You don't need to change the values of other fields but it will not be matter, so I approved it.

Reply:
Expect you would say so. ;)
I admit was a bit paranoid there.